### ISSUE TITLE
v0.2.1 breaks imports in vue3 ts component 

### ISSUE THREAD

**Issue by mikouaji** [wheatjs/vite-plugin-vue-type-imports#29]
After upgrading to v0.2.1 every component that makes use of the plugin has broken imports.
I don't know what exactly is responsible for this behavior, there are many other plugins in the project.
If you want to I can try to prepare a project that reproduces the problem when I have a bit of free time.

Reverting to v0.2.0 fixes the problem.

**Comment by Zolyn** [comment]
Can you provide a reproduction link/repo?

**Comment by mikouaji** [comment]
I'll try making one over the weekend

**Comment by ppierre** [comment]
https://github.com/ppierre/test-bug-import-plugin-vue-type-import

`runtime-core.esm-bundler.js:38 [Vue warn]: Failed to resolve component: HeartComp`

Just downgrading in the `packages.json`"solve" the problem:
```diff
-    "vite-plugin-vue-type-imports": "0.2.1"
+    "vite-plugin-vue-type-imports": "0.2.0"
```

Sorry that I can't be more helpfull this plugin is realy usefull. But whitout any error trace I don't know where to look.

**Comment by Zolyn** [comment]
It will be fixed in the next release.

**Comment by Zolyn** [comment]
Should be fixed in [v0.2.2](https://github.com/wheatjs/vite-plugin-vue-type-imports/releases/tag/v0.2.2)

**Comment by mikouaji** [comment]
Updating to 0.2.2 seems to work OK in my case
